frontend/app/server.tsx

```diff
@@ -19,6 +19,7 @@ import {
     extractConfigMeta,
     extractGAMeta,
 } from '@frontend/lib/parse-capi';
+import { extractScripts } from '../amp/components/lib/AMPScripts';
```

Comment:
I think we're trying to avoid `..` in imports now (use @frontend/..) instead. But we should probably add a lint rule on this.

Reply:
nice catch
also something to fix auto imports I guess